src/middlewared/middlewared/plugins/smb_/smbconf/reg_global_smb.py

```diff
@@ -86,13 +86,12 @@ def log_level_transform(entry, conf):
 
     def set_log_level(entry, val, data_in, data_out):
         loglevelint = int(LOGLEVEL_MAP.inv.get(val, 1))
```

Comment:
So, we **silently** default to MINIMUM if an invalid `val` is supplied?  Hmm.

Reply:
Old behavior. Validation occurs earlier.